src/components/Campaigns/CampaignAgeFilter.tsx

```diff
@@ -1,17 +1,17 @@
 import { FormControlLabel, Switch, Typography } from "@mui/material";
 import _ from "lodash";
 import moment from "moment";
-import { Dispatch } from "react";
+import { useContext } from "react";
```

Comment:
Is there really no other way we can do this without having to resort to pushing this part of the filter in to context?

Reply:
I'd be very happy to just refetch all active queries in cases like this where it's tricky to work out the full set of dependencies: https://www.apollographql.com/docs/react/data/refetching/#refetching-all-queries, would that work instead of pushing to context?